src/Microsoft.Extensions.Logging.EventSource/EventSourceLogger.cs

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Threading;
+using Newtonsoft.Json;
+using System.IO;
```

Comment:
(Apply to all files, as necessary. Also, ensure "System" namespaces are at the top - VS has a setting for this.)